bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -103,6 +105,8 @@ public final void onSuccess(Object message) {
           result =
               Adapters.ROW_ADAPTER.adaptResponse(
                   (com.google.cloud.bigtable.data.v2.models.Row) message);
+        } else if (message instanceof Result) {
+          result = (Result) message;
```

Comment:
Shouldnt this always be a Result? when would we see FlatRow or Row?

Reply:
As of this change, we have yet to migrate `BufferedMutatorHelper` with new wrappers, 

https://github.com/googleapis/java-bigtable-hbase/blob/fafa313acbe30374346b8264297edbea4b73d8d1/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java#L191-L193

so it is a possibility that it may call readModifyWriteRow() for mutation.
https://github.com/googleapis/java-bigtable-hbase/blob/fafa313acbe30374346b8264297edbea4b73d8d1/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutatorHelper.java#L205-L210

Reply:
got it, thanks